src/ecrc-frontend/src/components/base/header/__snapshots__/Header.test.js.snap

```diff
@@ -16,14 +16,14 @@ exports[`Header Component Matches the snapshot 1`] = `
         alt="B.C. Government Logo"
         className="img-fluid d-none d-md-block"
         height="44"
-        src="/criminalrecordcheck/images/bcid-logo-rev-en.svg"
+        src="undefined/images/bcid-logo-rev-en.svg"
```

Comment:
this seems interesting, did you figure out why this was happening?

Reply:
It appears to be happening because when Jest runs the tests, it does not set up the env var used to dynamically set the src, and so when it renders the page for a snapshot, that env does not exist and is undefined. It is kind of expected and I am not sure how far to look into this to try and solve it. When it runs locally in webpack the homepage in package.json gets loaded in as the PUBLIC_URL env var, so it load properly, in storybook PUBLIC_URL gets loaded with "." so it finds the images correctly.

Reply:
not a blocker for now, let's get this change in as is